src/index.html

```diff
@@ -3,24 +3,62 @@
   <head>
     <meta charset='UTF-8'>
     <title>Whats Cookin</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
   </head>
   <body>
     <header class='main-header'>
-      <h1>Whats Cookin</h1>
+      <h1>What's Cookin'</h1>
       <p class='user-info'>Hi, <span class='user-name'>user</span></p>
-      <button class='search-button' type='button'>
-        <img class='search-button' src="https://image.flaticon.com/icons/svg/149/149220.svg" alt="Search button">
-      </button>
-      <input type='text' placeholder='Search saved recipes by name or ingredient' value=''>
+      <div class='search-bar'>
+        <!-- <button class='search-button' type='button'> -->
+          <img class='search-image' src="https://image.flaticon.com/icons/svg/660/660029.svg" alt="Search button">
+        <!-- </button> -->
```

Comment:
From your HTML, it looks like you decided to remove the button tags and create buttons from images instead, that right?

Reply:
Yes. I think in the end it will work better to remove the button tags. I was running into buggy behavior when resizing the screen when I had the button tags wrapped around the image.